js/index.js

```diff
@@ -1 +1,99 @@
 // Your code goes here
+//
+const backGround = document.querySelector('.home');
+const funInSun = document.querySelectorAll('.destination');
+const buttonText = document.querySelector('.btn');
+const popUp = document.querySelector('.popup-ad');
+const welcomeText = document.querySelector('.container .intro p');
+const boatImage = document.querySelector('.content-destination img');
+const headerLinks = document.querySelectorAll('.nav a');
+const bodyColor = document.querySelector('.home');
+const funBus = document.querySelector('.intro h2');
+function hideThing(e) {
+  if(e.currentTarget.style.visibility === 'hidden') {
+    e.currentTarget.style.visibility = 'visible';
+  } else {
+    e.currentTarget.style.visibility = 'hidden';
+  }
+}
+
+headerLinks.forEach((item) => {
+  item.addEventListener('click', (e) => {
+    popUp.style.visibility = 'visible';
+  })
+});
+
+funBus.addEventListener('click', (e) => {
+  if( e.target.parentNode.style.background === 'gray') {
+    e.target.parentNode.style.background = 'blue';
+  } else {
+    e.target.parentNode.style.background = 'gray';
+  }
+});
+
+const pTags = document.querySelectorAll('p');
+pTags.forEach((item) => {
+  item.addEventListener('mouseover', hideThing);
+  item.addEventListener('mouseleave', hideThing);
+});
+// console.log(headerLinks);
+// 1
+boatImage.addEventListener('drag', (event) => {
+  boatImage.style.borderRadius = '1000px';
+});
+// 2
+boatImage.addEventListener('dragend', (event) => {
+  boatImage.style.borderRadius = '50px';
+});
+// 3
+welcomeText.addEventListener('dblclick', (event) => {
+  welcomeText.style.background = 'yellow';
+});
+// 4
+popUp.addEventListener('click', (event) => {
+  console.log('here is the pop up ad');
+  popUp.classList.add('display-none');
+});
+// 5
+funInSun[0].addEventListener('mouseover', (event) => { 
+  backGround.style.background = 'yellow';
+  document.querySelector('.intro h2').style.color = 'black';
+});
+//6
+funInSun[1].addEventListener('mouseover', (event) => { 
+  // console.log('we are over the div') 
+  backGround.style.background = 'purple';
```

Comment:
I love how the entire background color changes when you hover over each paragraph 😍